bin/run-browser-tests-ci

```diff
@@ -8,6 +8,7 @@ source bin/lib.sh
 docker::set_project_name_browser_tests
 
 docker volume create browser-tests-node-modules
+
```

Comment:
(Super nit- feel free to ignore) Probably better to skip this white space addition, since there's no other changes to this file